synth-converter/src/graph/utils.rs

```diff
@@ -0,0 +1,20 @@
+use crate::graph::namespaces::EX;
+use uuid::Uuid;
+use sophia_api::term::{bnode_id::BnodeId, SimpleTerm};
+use sophia_api::prelude::*;
+
+pub fn generate_bnode_term() -> SimpleTerm<'static> {
```

Comment:
question: do you know how SimpleTerm works internally, with the `Mown` string?
This made me puzzled haha. But I think `'static' is the only viable option.

Reply:
To be honest, I struggle a bit to understand the differences between the term implementations in Sophia. `SimpleTerm` just seems to me to work in way that I expect, which is less true for `RS_Term`